EX1/HW6.py

```diff
@@ -0,0 +1,208 @@
+import time
+import os
+import numpy as np
+from PIL import Image
+
+
```

Comment:
Documantation :

“Code tells you how; Comments tell you why.”

— [Jeff Atwood]

https://realpython.com/documenting-python-code/#commenting-vs-documenting-code